config/catalog.go

```diff
@@ -38,13 +38,29 @@ var (
 	catalogBlockReg = regexp.MustCompile(fmt.Sprintf(hclBlockRegExprFmt, MetadataCatalog))
 )
 
+// CatalogConfig represents the configuration for the Terragrunt catalog.
 type CatalogConfig struct {
-	DefaultTemplate string   `hcl:"default_template,optional" cty:"default_template"`
-	URLs            []string `hcl:"urls,attr" cty:"urls"`
+	// DisableShell disables shell functions in scaffold templates (CLI flags take precedence)
```

Comment:
Let's avoid having these diverge, @pseudomorph .

Can we name these `NoShell` (and `no_shell`) and `NoHooks` (and `no_hooks`), just like the CLI flags? That will make it easier for folks to guess what they're called, and how they're overridden in the CLI.